pinot-core/src/main/java/org/apache/pinot/core/realtime/impl/RealtimeSegmentConfig.java

```diff
@@ -40,13 +40,25 @@
   private final RealtimeSegmentStatsHistory _statsHistory;
   private final SegmentPartitionConfig _segmentPartitionConfig;
   private final boolean _aggregateMetrics;
+  private final boolean _nullHandlingEnabled;
 
   private RealtimeSegmentConfig(String segmentName, String streamName, Schema schema, int capacity,
       int avgNumMultiValues, Set<String> noDictionaryColumns, Set<String> varLengthDictionaryColumns,
       Set<String> invertedIndexColumns, RealtimeSegmentZKMetadata realtimeSegmentZKMetadata,
       boolean offHeap, PinotDataBufferMemoryManager memoryManager,
       RealtimeSegmentStatsHistory statsHistory, SegmentPartitionConfig segmentPartitionConfig,
       boolean aggregateMetrics) {
+    this(segmentName, streamName, schema, capacity, avgNumMultiValues, noDictionaryColumns, varLengthDictionaryColumns,
+        invertedIndexColumns, realtimeSegmentZKMetadata, offHeap, memoryManager, statsHistory, segmentPartitionConfig,
+        aggregateMetrics, false);
+  }
+
+  private RealtimeSegmentConfig(String segmentName, String streamName, Schema schema, int capacity,
```

Comment:
You don't need two constructors as it is private and always constructed from the builder.

Reply:
Done